packages/sw-background-sync-queue/demo/sw.js

```diff
@@ -16,20 +24,17 @@ let bgQueue = new goog.backgroundSyncQueue.BackgroundSyncQueue({callbacks:
 	},
 });
 
+const requestWrapper = new goog.runtimeCaching.RequestWrapper({
+  behaviors: [bgQueue],
+});
 
-self.addEventListener('fetch', function(e) {
-	if (e.request.url.startsWith('https://jsonplaceholder.typicode.com')) {
-		const clone = e.request.clone();
-		e.respondWith(fetch(e.request).catch((err)=>{
-			bgQueue.pushIntoQueue({
-				request: clone,
-			});
-			throw err;
-		}));
-	}
+const route = new goog.routing.ExpressRoute({
+  path: '/*',
+  origin: 'https://jsonplaceholder.typicode.com',
```

Comment:
That's not going to work as expected—`origin` isn't a supported parameter.

As of https://github.com/GoogleChrome/sw-helpers/pull/158, you can't use `ExpressRoute` for cross-origin routes. You should use `RegExpRoute` instead:

```js
const route = new goog.routing.RegExpRoute({
  regExp: new RegExp('^https://jsonplaceholder.typicode.com'),
  handler: new goog.runtimeCaching.NetworkOnly({requestWrapper}),
});
```